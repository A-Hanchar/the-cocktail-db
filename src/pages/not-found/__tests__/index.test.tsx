import {render, screen} from '@testing-library/react';

import NotFound from '../';

describe('<NotFound />', () => {
	it('SHOULD match snapshot', () => {
		const {asFragment} = render(<NotFound />);

		expect(asFragment()).toMatchSnapshot();
	});

	it('SHOULD renders the not found message', () => {
		render(<NotFound />);

		const messageElement = screen.getByTestId('notFoundMessage');

		expect(messageElement).toHaveTextContent('Page is not Found');
	});
});
