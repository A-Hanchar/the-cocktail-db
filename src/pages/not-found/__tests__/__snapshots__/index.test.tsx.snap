// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> SHOULD match snapshot 1`] = `
<DocumentFragment>
  <div
    class="box container"
    data-test-id="box"
  >
    <h1
      class="ui-text color-base size-36px"
      data-test-id="notFoundMessage"
    >
      Page is not Found
    </h1>
    <img
      alt="Page not Found"
      src="https://img.freepik.com/free-vector/oops-404-error-with-broken-robot-concept-illustration_114360-5529.jpg"
    />
  </div>
</DocumentFragment>
`;
